src/index.scss

```diff
@@ -30,346 +37,268 @@ $mobile-width: 750px;
-	height: 175px;
-	margin-left: 50px;
-
-	@include tablet {
-		height: 150px;
-		margin-left: 40px;
-	}
-
-	@include mobile {
-		/*68 px is the current height of the nav bar
-		will think of a better way to structure this later*/
-		width: default;
-		height: max(68px, calc(35px + 10vh));
-		margin-left: 15px;
-	}
+html, body, #root, #app {
+	height: 100%;
 }
 
-.navigation {
-	width: 100%;
+nav {
+	position: fixed;
+	z-index: 1;
 
-	font-size: 15px;
-	font-weight: 500;
+	width: 100vw;
+	height: 5vw;
 
-	color: $blue;
-	background-color: white;
+	background-color: transparentize(white, 0.95);
+	
+	#logo {
+		content:url('./bh7-assets/logo.svg');
+		height: 100%;
+		margin: 0vw 2vw;
+	}
 
 	ul {
-		list-style-type: none;
-		float: right;
-
-		margin: 0;
-		padding: 0;
-
-		li {
-			padding: 25px 25px;
-			text-align: center;
-			text-decoration: none;
+		display: flex;
+		align-items: center;
 
-			a {
-				color: $blue;
+		list-style-type: none;
 
-				border-bottom-style: solid;
-				border-color: transparentize($blue, 1);
-				transition: border-color 0.2s;
+		height: 100%;
+		padding-inline-start: 0;
 
-				&:hover {
-					border-color: transparentize($blue, 0);
-					transition: border-color 0.2s;
-				}
-			}
-		}
-	}
+		a {
+			display: flex;
+			align-items: center;
 
-	@include tablet {
-		display: inline-block;
-		
-		font-size: 12px;
-		font-weight: 400;
+			height: 100%;
+			margin: 0vw 2vw;
 
-		color: white;
-		background-color: $blue;
+			color: white;
 
-		ul li {
-			padding: 25px 15px;
-			a {
-				color: white;
-				border-bottom-style: solid;
-				border-color: transparentize(white, 1);
+			font-size: 1.25vw;
+			font-weight: 400;
 
-				&:hover {
-					border-color: transparentize(white, 1);
-				}
-				&:active {
-					border-color: transparentize(white, 0);
-				}
+			transition: font-weight 0.1s, color 0.2s, border-color 0.3s;
+			
+			&:hover {
+				font-weight: 600;
+				color: $orange;
+				border-color: transparentize($orange, 0);
 			}
-		}
+		}	
 	}
 
-	@include mobile {
-		/*no scaling for very short devices,
-		scaling matching the mlh badge scaling for avg devices,
-		high scaling for very tall devices
-		*/
-
-		margin-bottom: max(1px, calc(10vh - 40px), calc(30vh - 190px));
+	#mlh-badge {
+		position: absolute;
+		top: 0;
+		height: 13.5vw;
+		right: 2vw;
 	}
-}
 
-.section {
-	height: 100%;
-	display: flex;
-	flex-direction: column;
-	
-	@include desktop {
-		min-height: 750px;
-	}
-	@include tablet {
-		height: calc(100% - 74.5px);
-	}
 	@include mobile {
-		height: 0%;
-	}
-}
-
-.content {
-	width: 100%;
-	
-	max-width: 1050px;
-	text-align: center;
-	margin: auto;
-
-	@include tablet {
-		max-width: 100%;
-		flex-direction: column;
-		margin: auto;
-	}
-}
-
-.right-half {
-	flex: 50%;
-}
-.left-half {
-	@extend .right-half;
-	margin-bottom: 2.5vh;
-
-	@include tablet {
-		margin-bottom: 15vh;
+		background-color: transparentize(white, 0.85);
+		height: 20vw;
+		
+		#mlh-badge {
+			top: 0;
```

Comment:
nit: `top: 0` redundant

Reply:
It isn't redundant since I put the badge after the <ul> that holds the nav elements, but I've re-ordered it so I can remove this line.